uapi/src/systypes.rs

```diff
@@ -245,6 +245,7 @@ pub type StreamHandle = u32;
 /// As multiple events can be set at once, event field is using a
 /// bitfield model to keep C+Rust usage easy
 #[repr(C)]
+#[derive(Clone,PartialEq,Copy,Debug)]
```

Comment:
The file starts being heavy, with 1350 LOC ...

Reply:
Yes, because Rust require looonnngg writing in comparison with other languages.
An important part of this module SLOC.... are huge line-consuming test suites (repr(C) checks & so on).
This can be enhanced by moving tests to dedicated test/ directory, but the official way is based on src/ based tests definition : https://doc.rust-lang.org/book/ch11-01-writing-tests.html